posthog/hogql_queries/insights/funnels/funnel_event_query.py

```diff
@@ -85,16 +90,17 @@ def _aggregation_target_expr(self) -> ast.Expr:
         elif funnelsFilter.funnelAggregateByHogQL and funnelsFilter.funnelAggregateByHogQL != "person_id":
             aggregation_target = parse_expr(funnelsFilter.funnelAggregateByHogQL)
 
-        # TODO: is this still relevant?
-        # # Aggregating by Distinct ID
-        # elif self._aggregate_users_by_distinct_id:
-        #     aggregation_target = f"{self.EVENT_TABLE_ALIAS}.distinct_id"
-
         if isinstance(aggregation_target, str):
             return ast.Field(chain=[aggregation_target])
         else:
             return aggregation_target
 
+    def _aggregation_target_filter(self) -> ast.Expr | None:
+        if self._aggregation_target_expr() == ast.Field(chain=["person_id"]):
+            return None
+
+        return parse_expr("aggregation_target != '' and aggregation_target != null")
```

Comment:
just double checking `''` empty string is not a valid group value?
just in case it's valid -> maybe we show it on the UI as `(empty)` or something, we do that for retention